.github/ISSUE_TEMPLATE/1_bug_report_form.yml

```diff
@@ -17,6 +17,12 @@ body:
       description: A concise description of what issue you're experiencing. **Example:** "When I click on the 'Sign in' button, I expect to be redirected to the login page, but instead I get a 404 error."
     validations:
       required: true
+  - type: textarea
+    attributes:
+      label: URL
```

Comment:
Make it clear what URL is expected here as well. (URL where the bug has occurred etc..)

Reply:
Done